extensions/amp-form/0.1/test/test-amp-form.js

```diff
@@ -2896,6 +2897,57 @@ describes.repeated(
             }
           );
         });
+
+        describe('Form Dirtiness State', () => {
+          let form, ampForm, input;
+
+          function changeInput(element, value) {
```

Comment:
nit: didn't check whole AMP runtime but this file keeps helper methods after beforeEach and afterEach